vsintegration/src/FSharp.Editor/CodeLens/AbstractCodeLensDisplayService.fs

```diff
@@ -47,7 +46,7 @@ type CodeLensDisplayService (view : IWpfTextView, buffer : ITextBuffer, layerNam
     /// Text view for accessing the adornment layer.
     member val View: IWpfTextView = view
 
-    member val CodeLensLayer = view.GetAdornmentLayer layerName
+    member val CodeLensLayer = view.GetAdornmentLayer "LineLens"
```

Comment:
I am sure you know it, but I do not like having CodeLens and LineLens at the same time, now that we know we only support 1 mode.
Pick a single name and stick to that.

Reply:
(rather a suggestin for next step, definitely do merge this in as it is)

Reply:
I wanted to do this in a followup. It will take us some time to figure out branding, I'll probably just temporarily have Lens :)